asllib/tests/stdlib.t/set-bits.asl

```diff
@@ -0,0 +1,36 @@
+func main () => integer
+begin
+  assert HighestSetBit{0}('') == -1;
+  assert LowestSetBit{0}('') == 0;
+  assert HighestSetBit{1}('0') == -1;
+  assert LowestSetBit{1}('0') == 1;
+  assert HighestSetBit{1}('1') == 0;
+  assert LowestSetBit{1}('1') == 0;
+
+  for n = 2 to 10 do
+    assert HighestSetBit{n}(Zeros{n}) == -1;
+    assert LowestSetBit{n}(Zeros{n}) == n;
+
+    // This enumerates all non-zeros bitvectors of length n
```

Comment:
I'm not sure this is true - for e.g. `n = 3` I seem to get only `100`, `010`, `110`